src/dbConfig.ts

```diff
@@ -0,0 +1,16 @@
+import { ConnectionConfiguration } from 'tedious';
+
+export const dbConfig: ConnectionConfiguration = {
+    server: 'localhost',
+    options: {
+        trustServerCertificate: true,
+        database: 'bookish',
+    },
+    authentication: {
+        type: 'default',
+        options: {
+            userName: 'admin',
+            password: 'b00kish_4dmin!',
```

Comment:
Just to flag that in the real world we wouldn't commit this; instead this should be an **environment variable**.

If you'd like to try this now you've got some other practice in, that would be a good thing to do :)